docs/standard/base-types/enumeration-format-strings.md

```diff
@@ -26,13 +26,13 @@ You can use the <xref:System.Enum.ToString%2A?displayProperty=fullName> method t
 |Format string|Result|  
 |-------------------|------------|  
 |G or g|Displays the enumeration entry as a string value, if possible, and otherwise displays the integer value of the current instance. If the enumeration is defined with the **Flags** attribute set, the string values of each valid entry are concatenated together, separated by commas. If the **Flags** attribute is not set, an invalid value is displayed as a numeric entry. The following example illustrates the G format specifier.<br /><br /> [!code-csharp[Formatting.Enum#1](../../../samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs#1)]
- [!code-vb[Formatting.Enum#1](../../../samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb#1)]|  
+| |[!code-vb[Formatting.Enum#1](../../../samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb#1)]|
```

Comment:
... and now that I think about it \<pre> is a block-level element, so we don't need a \<br> at all. It will render like this (hacked via Google Dev Tools) ...

![capture](https://user-images.githubusercontent.com/1622880/27008333-93fd3596-4e34-11e7-9e75-d43f1cf4279e.PNG)